community/src/Config/config.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config()
+
+const connectionParams = {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+}
+
+const config = {
+  port: process.env.PORT || 8003,
+  mongoURI: process.env.DB,
+  connectionParams
+}
```

Comment:
you can more this part to `community/connectToDB.js` instead and let you database is being managed in one place, no need to have a database configer if your configs are quite minimum